wpilibc/src/main/native/cpp/Joystick.cpp

```diff
@@ -252,27 +233,21 @@ bool Joystick::GetTriggerReleased() {
  *
  * @return The state of the top button.
  */
-bool Joystick::GetTop() const {
-  return GetRawButton(static_cast<int>(Button::kTop));
-}
+bool Joystick::GetTop() const { return GetRawButton(Button::kTop); }
 
 /**
  * Whether the top button was pressed since the last check.
  *
  * @return Whether the button was pressed since the last check.
  */
-bool Joystick::GetTopPressed() {
-  return GetRawButtonPressed(static_cast<int>(Button::kTop));
-}
+bool Joystick::GetTopPressed() { return GetRawButtonPressed(Button::kTop); }
 
 /**
  * Whether the top button was released since the last check.
  *
  * @return Whether the button was released since the last check.
  */
-bool Joystick::GetTopReleased() {
-  return GetRawButtonReleased(static_cast<int>(Button::kTop));
-}
+bool Joystick::GetTopReleased() { return GetRawButtonReleased(Button::kTop); }
```

Comment:
You don't even need `Button::` qualifier (nor the `Axis::` one), but I guess it doesn't hurt.

Reply:
I wish enum required them to be honest.

Reply:
well, that's what `enum class` does, but that has other downsides.